internal/aez/general.go

```diff
@@ -83,3 +104,41 @@ func (stat *STAT) Time(epoch time.Time) time.Time {
 func (stat *STAT) Nanoseconds() int64 {
 	return ccsds.UnsegmentedTimeNanoseconds(stat.TS, stat.TSS)
 }
+
+// CSVSpecifications returns the version of the spec used
+func (stat STAT) CSVSpecifications() []string {
+	return []string{"AEZ", Specification}
+}
+
+// CSVHeaders returns the header row
+func (stat STAT) CSVHeaders() []string {
+	var headers []string
+	headers = append(headers, "STATTIME", "STATNANO")
+	val := reflect.Indirect(reflect.ValueOf(stat))
+	t := val.Type()
+	for i := 0; i < t.NumField(); i++ {
+		name := t.Field(i).Name
+		if name != "TS" && name != "TSS" {
+			headers = append(headers, t.Field(i).Name)
+		}
+	}
+	return headers
+}
+
+// CSVRow returns the data row
+func (stat STAT) CSVRow() []string {
+	var row []string
+	gpsTime := time.Date(1980, time.January, 6, 0, 0, 0, 0, time.UTC)
+	statTime := stat.Time(gpsTime)
+	row = append(row, statTime.Format(time.RFC3339Nano), fmt.Sprintf("%v", stat.Nanoseconds()))
+	val := reflect.Indirect(reflect.ValueOf(stat))
+	t := val.Type()
+	for i := 0; i < val.NumField(); i++ {
+		name := t.Field(i).Name
+		if name != "TS" && name != "TSS" {
```

Comment:
vet inte vad dessa är (antar det är tiden), underlättar kanske med en kommentar

Reply:
Yes det är TimeSeconds och TimeSubSeconds. Lägger in en kommentar.